fleet/agent_group_service.go

```diff
@@ -159,7 +159,7 @@ func (svc fleetService) CreateAgentGroup(ctx context.Context, token string, s Ag
 	err = svc.addAgentsToAgentGroupChannel(token, ag)
 	if err != nil {
 		// TODO should we roll back?
-		svc.logger.Error("error adding agents to group channel", zap.Error(errors.Wrap(ErrMaintainAgentGroupChannels, err)))
+		svc.logger.Error("error adding agents to group channel", zap.String("agent_group_id", ag.ID), zap.Error(errors.Wrap(ErrMaintainAgentGroupChannels, err)))
```

Comment:
sorry one more: just "group_id" on this one to match the operational metrics we made